src/Compilers/VisualBasic/Test/Emit/ErrorHandling.vb

```diff
@@ -1037,6 +1037,51 @@ End]]>)
 ]]>)
         End Sub
 
+        <Fact>
+        Public Sub SynthesizingDefaultConstructorsWithMsCorLibMissing_Nothing()
+            Dim comp = CompilationUtils.CreateCompilation("
+Namespace System
+    Public Class Object
+    End Class
+End Namespace
+Class Test
+End Class
+")
+
+            CompilationUtils.AssertTheseDiagnostics(comp, <expected>
+BC30002: Type 'System.Void' is not defined.
+    Public Class Object
+    ~~~~~~~~~~~~~~~~~~~~
+BC30183: Keyword is not valid as an identifier.
```

Comment:
This error is likely preventing us from testing the scenario we are interested in.